src/RemarkableFS.php

```diff
@@ -162,7 +162,12 @@ public function typeToIcon($type)
      */
     protected function calcPath($id)
     {
-        if (!isset($this->index[$id])) throw new \Exception("Unknown ID $id. Inconsitent meta data");
+    	if($id == "trash")
+    		return "";
```

Comment:
Here's a proposed different fix. Later, change as follows:
```php
         if ($item['Parent'] === '') {
             // top level item
             $item['Path'] = '/' . $item['VissibleName'];
+        } else {if ($item['Parent'] === 'trash') {
+           // trash item
+           $item['Path'] = 'trash/' . $item['VissibleName'];
         } else {
             // recursion
             $item['Path'] = $this->calcPath($item['Parent']) . '/' . $item['VissibleName'];
-        }
+        }}

This doesn't actually work (though it stops crashing on my machine), but the point is to return a sensible path rather than an empty path.
(Unless the empty path is actually correct --- I'm not an expert in this API, just proposing a slightly different way to do this.)
```

Reply:
No problem, I just needed a solution that works and I thougt I'd share it if someone is interested :blush: I don't know either what's the right solution here :woman_shrugging: